toonz/sources/CMakeLists.txt

```diff
@@ -137,7 +158,7 @@ if(WIN32)
         set(QT_LIB ${SDKROOT}/quicktime/QT73SDK/Libraries/QTMLClient.lib)
     endif()
 else()
-    find_library(GLUT_LIB GLUT)
+    find_library(GLUT_LIB glut)
     find_library(GL_LIB OpenGL)
```

Comment:
had to modify `find_library(GL_LIB OpenGL)` to `find_library(GL_LIB GL)` on Ubuntu 15.10 to not give any errors.